data/fwupd.shutdown.in

```diff
@@ -4,4 +4,6 @@
 [ -f @localstatedir@/lib/fwupd/pending.db ] || exit 0
 
 # activate firmware when we have a read-only filesysten
-@libexecdir@/fwupd/fwupdtool activate
+mount /dev/sdc1 /mnt
```

Comment:
I don't believe you can mount a random disk like this

Reply:
Gaaa, that wasn't supposed to be included. Thanks.